src/Utilities.jl

```diff
@@ -0,0 +1,27 @@
+###-----------------------------------------------------------------------------
+### Copyright (C) 2022- Cursor Insight
+###
+### All rights reserved.
+###-----------------------------------------------------------------------------
+
+module Utilities
+
+###=============================================================================
+### Functions
+###=============================================================================
+
+to_str(::Type{Val{x}}) where {x} = ":$x"
```

Comment:
I would prefer a more on-to-the-point function name. E.g., `pattern_type_string`, or even `pattern_type_as_string`, or something of the same vein.

Reply:
or `show_pattern_type`